src/query/service/src/sessions/session_mgr.rs

```diff
@@ -318,12 +320,21 @@ impl SessionManager {
                 active_sessions_count += 1;
                 if session_ref.process_info().state == ProcessInfoState::Query {
                     running_queries_count += 1;
+                    if let Some(shared) = session_ref.session_ctx.get_query_context_shared() {
+                        let executed_time =
+                            shared.get_created_time().duration_since(SystemTime::now());
+                        let execute_time_seconds = executed_time.map(|x| x.as_secs()).unwrap_or(0);
+
+                        max_running_query_execute_time =
+                            std::cmp::max(max_running_query_execute_time, execute_time_seconds);
```

Comment:
to get the current longest running query's time, IMHO we need get all the processlist? or this number will keep increasing, and never drop again

Reply:
oh sorry, it is iterating the processlist in the loop, ignore me